osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -475,6 +475,8 @@ private void postMessage(TextBox textbox, bool newText)
 
             if (target == null) return;
 
+            currentChannelContainer.Child.ScrollToEnd();
```

Comment:
Is there any reason why this isn't done internal to DrawableChannel? Maybe via `NewMessagesArrived`?

Reply:
Ah yes, this could be a better solution. I think we can tell using type-checking whether it's a local message.